DangerSwap/DangerSwap/Services/Currency/CurrencyService.cs

```diff
@@ -28,6 +28,7 @@ public async Task UploadCurrencies(bool isFiat)
                     } else
                     {
                         await UpdateCurrency(originCurrency, item);
+                        //
```

Comment:
Row 20. UploadCurrencies could use Guard Clause. Method name "UploadCurrencies" doesn't represent the real action of the method